docs/getting-started/detections-req.asciidoc

```diff
@@ -35,6 +35,65 @@ of at least 32 characters. For example:
 IMPORTANT: After changing the `xpack.encryptedSavedObjects.encryptionKey` value
 and restarting {kib}, you must restart all detection rules.
 
+[discrete]
+[[enable-and-access-detections]]
+== Enable and Access Detections
+Enabling the Detections feature allows you to view and use the Detections page from within the Security app. You only need to complete this action once and can choose between enabling it for a single Kibana space or across multiple ones. Once the detections feature is enabled, additional privileges are needed to use Detection rules and alerts, described in the following table.
+
+NOTE: For instructions about using Machine Learning jobs and rules, refer to <<ml-requirements, Machine learning job and rule requirements>>.
+
+[discrete]
+[width="100%",options="header"]
+|==============================================
+|Action |Cluster Privileges |Index Privileges |Kibana Privileges
+
+|Enable the Detections feature in your Kibana space
+|The `manage` privilege
+
+a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices where `<kib-space>` is the {kib} space name:
+
+* `.siem-signals-<kib-space>`
+* `.lists-<kib-space>`
+* `.items-<kib-space>`
+|{kib} space `All` privileges for the `Security` feature (see
+{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
+
+
+|Enable the Detections feature in all Kibana spaces
+|The `manage` privilege
+a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices:
+
+* `.siem-signals-*`
+* `.lists-*`
+* `.items-*`
+|{kib} space `All` privileges for the `Security` feature (see
+{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
```

Comment:
Ahhh, I think I see the distinction from above. Breaking out the `per space` vs `all space` access to alerts. This makes sense to me, but just note that even with the above `all space` configuration the user will need to go to the security app of each space to complete the `setup` of the detection engine (creating the index/ILM, etc).